spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerUtils.java

```diff
@@ -31,15 +32,13 @@
 	 */
 	protected static boolean isWindows() {
 		String osName = null;
+
 		try {
 			osName = System.getProperty("os.name");
-		} catch (Exception e) {
-		}
-		if (osName == null) {
-			return false;
 		}
-		else {
-			return osName.toLowerCase().startsWith("windows");
+		catch (Exception e) {
 		}
```

Comment:
If you intend to not do anything on this exception, then perhaps a `// ignore` comment in the catch, just ti communicate that its intentional

Reply:
Also good point.  I usually name the variable `ignore`...I'll update.